shed/event_streams.py

```diff
@@ -300,11 +309,10 @@ def start(self, docs):
             The document
         """
         self.run_start_uid = str(uuid.uuid4())
+        self.parent_uids = [doc['uid'] for doc in docs if doc]
```

Comment:
It combines two lines into one (since we are doing the testing in the comprehension). But in the long term I think they are pretty much the same (although I can change it back if you want have no real preference one way or another.) The only place I've seen this conditional invoked is in the `Query` nodes where we may not care about the input documents.